decoders-bencode.opam

```diff
@@ -11,15 +11,15 @@ doc: "https://mattjbray.github.io/ocaml-decoders/"
 bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
 depends: [
   "ocaml" {>= "4.03.0"}
-  "dune" {>= "2.0"}
+  "dune" {>= "2.8" & >= "2.0"}
```

Comment:
that seems redundant, and why bump the minimal version of dune?

Reply:
That change was auto generated by running `dune upgrade`. Revert it?

Reply:
Also, if you ever decide to compile it all with melange, the minimum version is 2.8. That will also get you the `let*` syntax available in bucklescript/rescript when its imported as a dep. If that is of any interest to you.

Reply:
personally I'd revert it, yes. If you use melange, you can add `dune >= 2.8` in your project, it doesn't mean each dependency has to have the same constraint :). Opam will account for them all.

Reply:
Apparently it was changing the dune version to `2.8` which automatically add the version to the dune files. So reverter `decoders.opam` dune version to what you had.